@xen-orchestra/lite/src/components/pool/network/PoolNetworksTable.vue

```diff
@@ -0,0 +1,234 @@
+<template>
+  <div class="container">
+    <UiTitle>
+      {{ $t('networks') }}
+      <template #actions>
+        <UiDropdownButton
+          v-tooltip="$t('coming-soon')"
+          disabled
+          :left-icon="faPlus"
+          variant="secondary"
+          accent="info"
+          size="medium"
+        >
+          {{ $t('new') }}
+        </UiDropdownButton>
+      </template>
+    </UiTitle>
+    <div class="content">
+      <UiQuerySearchBar class="table-query" @search="(value: string) => (searchQuery = value)" />
```

Comment:
Typing is already inferred here.

```suggestion
      <UiQuerySearchBar class="table-query" @search="value => (searchQuery = value)" />
```